deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -14,8 +14,24 @@ namespaceOverride: ""
 ##
 fullnameOverride: ""
 
+## Configuration for installing openebs-monitoring dashboards in already installed prometheus stack
+customPrometheusStack:
+  enabled: false
+
+  grafana:
+    ## Deploy custom openebs dashboards
+    customDashboardsEnabled: true
```

Comment:
Rather than `customDashboardsEnabled` can we use `openebsDashboardsEnabled`? If yes, then we need to make a change in line no: 124 also.